packages/calypso-e2e/src/browser-manager.ts

```diff
@@ -114,11 +67,19 @@ export async function launchBrowserContext(): Promise< BrowserContext > {
 	// By default, record video for each browser context.
 	const videoDir = getVideoDir();
 	const dimension = getScreenDimension();
+	const timestamp = getDateString();
+	const userAgent = `user-agent=Mozilla/5.0 (wp-e2e-tests) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/${ await browser.version() } Safari/537.36`;
 
 	// Generate a new BrowserContext.
 	return await browser.newContext( {
 		viewport: null, // Do not override window size set in the browser launch parameters.
 		recordVideo: { dir: videoDir, size: { width: dimension.width, height: dimension.height } },
+		userAgent: userAgent,
+		logger: {
+			isEnabled: ( _name ) => _name === 'api',
+			log: ( _name, _severity, _message ) =>
+				writeLog( { name: _name, severity: _severity, message: _message }, timestamp ),
```

Comment:
Do we need the underscore prefixes here?

Reply:
I had to add the underscore prefix here to placate the TypeScript compiler. Let me see if I can remove them without issues in my next revision.